cxplat/inc/winuser/cxplat_fault_injection.h

```diff
@@ -14,8 +14,10 @@ CXPLAT_EXTERN_C_BEGIN
  *
  * @param[in] stack_depth Number of stack frames to capture when a fault is
  * injected.
+ *
  * @retval CXPLAT_STATUS_SUCCESS The operation was successful.
  * @retval CXPLAT_STATUS_OUT_OF_MEMORY Operation failed due to memory allocation failure.
+ * @retval CXPLAT_STATUS_INVALID_STATE Operation failed due to invalid state (e.g. already initialized).
```

Comment:
```suggestion
 * @retval CXPLAT_STATUS_INVALID_STATE Operation failed due to invalid state (e.g., already initialized).
```
nit: always put comma after "i.e." or "e.g." (per Chicago Manual of Style)